components/prism-stylus.js

```diff
@@ -0,0 +1,61 @@
+Prism.languages.stylus = {
+	'comment': {
+		pattern: /(^|[^\\])(\/\*[\w\W]*?\*\/|\/\/.*?(\r?\n|$))/g,
+		lookbehind: true
+	},
+	'keyword': /(px|r?em)\b|\b(is|defined|not|isnt|and|or|unless|for|in)\b/g,
+	'atrule': /@[\w-]+(?=\s+\S+)/gi,
+	'url': /url\((["']?).*?\1\)/gi,
+	'variable': /^\s*([\w-]+)(?=\s*[+-\\]?=)/gm,
+	'string': /("|')(\\\n|\\?.)*?\1/g,
+	'important': /\B!important\b/gi,
+	'hexcode': /#[\da-f]{3,6}/gi,
+	'entity': /\\[\da-f]{1,8}/gi,
+	'number': /\d+\.?\d*%?/g,
+	'selector': [
+		{
+			pattern: /:{1,2}after|before|first-letter|first-line|selection/g,
```

Comment:
I think you forgot parentheses around the keywords alternatives.
Also, `:{1,2}` could be written `::?`.